recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: True  # [not linux]
-  number: 0
+  skip: True  # [win]
+  number: 1
```

Comment:
Build number has to be bumped because kubefed was not originally included int the linux build.